development/tools/jobs/incubator/compass/compass_integration_test.go

```diff
@@ -0,0 +1,144 @@
+package compass_test
+
+import (
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const compassIntegrationTestJobPath = "./../../../../../prow/jobs/incubator/compass/compass-integration.yaml"
+
+func TestCompassIntegrationVMJobsReleases(t *testing.T) {
+	// WHEN
+	unsupportedReleases := []tester.SupportedRelease{tester.Release11}
+
+	for _, currentRelease := range tester.GetKymaReleaseBranchesBesides(unsupportedReleases) {
+		t.Run(currentRelease, func(t *testing.T) {
+			jobConfig, err := tester.ReadJobConfig(compassIntegrationTestJobPath)
+			// THEN
+			require.NoError(t, err)
+			actualPresubmit := tester.FindPresubmitJobByName(jobConfig.Presubmits["kyma-incubator/compass"], tester.GetReleaseJobName("compass-integration", currentRelease), currentRelease)
+			require.NotNil(t, actualPresubmit)
+			assert.False(t, actualPresubmit.SkipReport)
+			assert.True(t, actualPresubmit.Decorate)
+			assert.Equal(t, "github.com/kyma-incubator/compass", actualPresubmit.PathAlias)
+			tester.AssertThatHasExtraRefTestInfra(t, actualPresubmit.JobBase.UtilityConfig, currentRelease)
+			tester.AssertThatHasExtraRefs(t, actualPresubmit.JobBase.UtilityConfig, []string{"cli"})
+			tester.AssertThatHasPresets(t, actualPresubmit.JobBase, tester.PresetGCProjectEnv, tester.PresetKymaGuardBotGithubToken, "preset-sa-vm-kyma-integration")
+			assert.False(t, actualPresubmit.AlwaysRun)
+			assert.Len(t, actualPresubmit.Spec.Containers, 1)
+			testContainer := actualPresubmit.Spec.Containers[0]
+			assert.Equal(t, tester.ImageKymaClusterInfra20190528, testContainer.Image)
+			assert.Len(t, testContainer.Command, 1)
+			assert.Equal(t, "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/provision-vm-compass.sh", testContainer.Command[0])
+			tester.AssertThatSpecifiesResourceRequests(t, actualPresubmit.JobBase)
+		})
+	}
+}
+
+func TestCompassIntegrationJobsPresubmit(t *testing.T) {
+	tests := map[string]struct {
+		givenJobName            string
+		expPresets              []tester.Preset
+		expRunIfChangedRegex    string
+		expRunIfChangedPaths    []string
+		expNotRunIfChangedPaths []string
+	}{
+		"Should contains the compass-integration job": {
```

Comment:
`Should contain` without `s`

![image](https://user-images.githubusercontent.com/7155799/60891649-ff46d080-a25d-11e9-9d54-e8027226edaa.png)